playgame.py

```diff
@@ -93,8 +94,10 @@ def determine_winner(results):
     return winners
 
 def run_one_game(args):
```

Comment:
Do you think the name of this function should be updated to reflect the fact that it can run multiple games for a single set of arguments? Or maybe a new function that calls this one in a loop?

```py
def run_one_configuration(args):
    for _ in range(args.iterations):
        run_one_game(args)
```

Reply:
The latter